package.json

```diff
@@ -37,5 +37,12 @@
   "homepage": "https://github.com/NYPL/scsb-rest-client#readme",
   "dependencies": {
     "p-limit": "^7.2.0"
-  }
+  },
+  "files": [
+    ".nvmrc",
+    "CHANGELOG.md",
+    "README.md",
+    "package.json",
+    "scsb-rest-client.js"
```

Comment:
Note, I believe README and package.json are [included by default](https://docs.npmjs.com/cli/v10/configuring-npm/package-json?v=true#files). And `.nvmrc` is probably not needed since it's presence in `node_modules` means nothing to the including app.